Source/NonVisuals/CockpitMaster/Panels/CDU737PanelBase.cs

```diff
@@ -226,7 +237,6 @@ public override List<string> ExportSettings()
             }
 
             var result = new List<string>();
```

Comment:
Just a totally personal preference but I prefer the syntax 
`List<string> result = new();`
Makes for easier read and reduce the `var` usage  (IMHO)

Reply:
done